### ISSUE TITLE
Sort by popularity

### ISSUE THREAD

**Issue by ecomkings** [christ0ph3r/cryptocurrency-dashboard#1]
Can you add the number of tweets about all the cryptocurrency coins? And have something like number of tweets in the last hour, last 24 hours, and create an alert when you see a spike in a number of tweets about any coin?

**Comment by Dfieb** [comment]
Is it somehow possible to attach the crypto chart of the selected coin? Otherwise great work! Thank you!

**Comment by ecomkings** [comment]
@christ0ph3r Yeah i was trying to figure out how I could sort the cryptocurrencies by number of tweets, I'm trying to find emerging  trends... like aeron for example/
